### PR TITLE
[terraform-0.11] add package series pinned to 0.11 major version

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add a package pinned to 0.11 series of terraform
* Install alternative into `/usr/local/terraform/0.11/bin/terraform` suitable for search paths (.e.g `PATH="/usr/local/terraform/0.11/bin:${PATH}"`)  

## why
* The reality of the terraform `0.11` -> `0.12` move is that we'll need to straddle both versions for some time

### CODE REVIEW

File: codefresh/apk.yml [hunk 1]
```diff
@@ -28,6 +28,9 @@ steps:
 
   docker_build:
     type: "parallel"
+    title: "Build Docker Images"
+    description: "Build Docker images for all recent alpine releases"
+    stage: Prepare
```
Comment (aknysh) [inline codefresh/apk.yml]: is this stage `Prepare` or `Build`?

Comment (osterman) [reply codefresh/apk.yml]: All things relative. We are preparing the docker images that will be used to build the alpine packages. The docker images are not the deliverable, the packages are. So building the images is just a preparatory step.

Comment (osterman) [reply codefresh/apk.yml]: image

File: vendor/terraform-0.11/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+pkgrel="${PACKAGE_RELEASE}"
+mjrver=${pkgver%.*}
+pkgname="terraform_${mjrver}"
+pkgdesc="${PACKAGE_DESCRIPTION}"
+arch="x86_64"
+url="https://releases.hashicorp.com/terraform/${pkgver}/terraform_${pkgver}_linux_amd64.zip"
+license="MPL-2.0"
+depends="dpkg"
+makedepends=""
+source="https://releases.hashicorp.com/terraform/${pkgver}/terraform_${pkgver}_linux_amd64.zip"
```
Comment (aknysh) [inline vendor/terraform-0.11/APKBUILD]: `source` is defined but not used?

Comment (osterman) [reply vendor/terraform-0.11/APKBUILD]: This is not a shell script, but an alpine package configuration file. 

See https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#source

**Pull request merged**